### PR TITLE
Default partition n

### PR DESCRIPTION
**Pull request by travisturner**

### CODE REVIEW

File: holder_test.go [hunk 1]
```diff
@@ -232,7 +232,7 @@ func (h *Holder) MustCreateFrameIfNotExists(index, frame string) *Frame {
 // MustCreateFragmentIfNotExists returns a given fragment. Panic on error.
 func (h *Holder) MustCreateFragmentIfNotExists(index, frame, view string, slice uint64) *Fragment {
 	idx := h.MustCreateIndexIfNotExists(index, pilosa.IndexOptions{})
-	f, err := idx.CreateFrameIfNotExists(frame, pilosa.FrameOptions{})
+	f, err := idx.CreateFrameIfNotExists(frame, pilosa.FrameOptions{CacheType: pilosa.CacheTypeRanked})
```
Comment (jaffee) [inline holder_test.go]: I did this when I was trying to debug - I assume most of these tests want ranked frames, but I guess there probably should be a way to set it.

File: server/server_test.go [hunk 2]
```diff
@@ -291,14 +291,14 @@ func TestMain_FrameRestore(t *testing.T) {
 
 	// Create frames.
 	client := m0.Client()
-	if err := client.CreateIndex(context.Background(), "x", pilosa.IndexOptions{}); err != nil && err != pilosa.ErrIndexExists {
+	if err := client.CreateIndex(context.Background(), "i", pilosa.IndexOptions{}); err != nil && err != pilosa.ErrIndexExists {
```
Comment (jaffee) [inline server/server_test.go]: What was going on with this test? I was getting an error like "index does not exist" - did you see that?

Comment (travisturner) [inline server/server_test.go]: yes, it was doing a restore on an empty node for an index that it didn't think it was responsible for. by changing the index name to `i`, the partition lined up with the node as the test expected.

File: holder_test.go (hunk 1 above)
Comment (travisturner) [inline holder_test.go]: oh, i see. i saw that and thought it was part of making the cache type default to "ranked". let me take a look.

**Pull request merged**